hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml

```diff
@@ -5426,4 +5426,36 @@
     <value>100ms</value>
   </property>
 
+  <property>
+    <description>
+      The interval at which the policy generator runs, default is one hour
+    </description>
+    <name>yarn.federation.gpg.policy.generator.interval-ms</name>
```

Comment:
Not sure how we handle this with backports but as the metric is general durations with units, it shouldn't include ms.

Reply:
Thank you for the reminder. I plan to implement this part according to the new specifications, excluding "ms". I will retain the old configuration and add a new configuration. During parsing, I will handle both configurations. If a user uses the old configuration, I will log a message to inform them.